services/apps/alcs/src/alcs/search/application/application-search-view.entity.ts

```diff
@@ -45,11 +37,6 @@ export class SearchApplicationSubmissionStatusType {
       .from(ApplicationSubmission, 'as2')
       .innerJoin(Application, 'a', 'a.file_number = as2.file_number')
       .withDeleted()
```

Comment:
I think this is no longer needed, since it applied to application type.

Reply:
Removed